app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,41 @@
+package ar.com.wolox.android.example.ui.login
+
+import android.widget.Toast
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentLoginBinding
+import ar.com.wolox.android.example.ui.viewpager.ViewPagerActivity
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+
+class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding, LoginPresenter>(), LoginView {
+
+    override fun init() {
+        presenter.checkUserSession()
+    }
+
+    override fun layout() = R.layout.fragment_login
+
+    companion object {
+        fun newInstance() = LoginFragment()
+    }
```

Comment:
Move the companion object to the very bottom of every class (just a convention!)